Cargo.toml

```diff
@@ -8,13 +8,13 @@ license = "Apache-2.0"
 description = "Jujutsu (an experimental VCS)"
 homepage = "https://github.com/martinvonz/jj"
 repository = "https://github.com/martinvonz/jj"
-documentation = "https://docs.rs/jujutsu"
+documentation = "https://docs.rs/jj-cli"
 readme = "README.md"
 keywords = ["VCS", "DVCS", "SCM", "Git", "Mercurial"]
 categories = ["command-line-utilities", "development-tools"]
 default-run = "jj"
 
-[[bin]]
+[[bin]] 
```

Comment:
There's an errant space at the end of the line.

Reply:
Fixed